vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -840,7 +860,7 @@ public void success(PinCodeInfo pinCodeInfo) {
                 VimeoClient.sContinuePinCodeAuthorizationRefreshCycle = true;
                 mPinCodeAuthorizationTimer.scheduleAtFixedRate(
                         new PinCodePollingTimerTask(pinCodeInfo, mPinCodeAuthorizationTimer,
-                                                    pinCodeInfo.getExpiresIn(), authCallback, sSharedInstance,
+                                                    pinCodeInfo.getExpiresIn(), authCallback, getInstance(),
```

Comment:
This change isn't functionally equivalent since `getInstance` might throw an error. I think it's fine - and probably even better - just don't know if we're making any assumptions elsewhere that will cause problems.